Numerical Hessian Code

bessvlai opened the pull request:
The implementation of the numerical Hessian and partial harmonic vibrational analysis (PHVA)

shiozaki commented:
Just as a record: There is one TODO to be fixed before this is merged into the master branch. Also perhaps it's good to include a test (which I can do myself before merging)

bessvlai commented:
@shiozaki I have checked the code and it is ready to merge.  We could include the following for testing. Perhaps also as an example in the manual. I believe I added keywords but I don't think I added an example.

{ "bagel" : [

{ 
  "title" : "molecule",
  "basis" : "svp",
  "df_basis" : "cc-pvdz-jkfit",
  "geometry" : [
  { "atom" : "H", "xyz" : [      0.676914,      2.781690,      3.773783 ] },
  { "atom" : "O", "xyz" : [      0.064239,      1.944715,      2.324074 ] },
  { "atom" : "H", "xyz" : [     -3.307624,      3.886786,      2.344879 ] },
  { "atom" : "H", "xyz" : [     -3.312992,      1.002931,      4.011981 ] },
  { "atom" : "C", "xyz" : [     -2.575001,      1.949136,      2.323306 ] },
  { "atom" : "C", "xyz" : [     -3.491636,      0.596669,     -0.021682 ] },
  { "atom" : "H", "xyz" : [     -2.798895,     -1.344513,     -0.045732 ] },
  { "atom" : "H", "xyz" : [     -2.795117,      1.545961,     -1.713429 ] },
  { "atom" : "H", "xyz" : [     -5.552636,      0.559870,     -0.086997 ] }
 ]
},

{ 
  "title" : "hessian",
  "partial" : true,
  "atom_list" : [1,2],
  "method" : [ {
     "title" : "hf"
  } ]
}

]}
